test-stack/docker-compose.yml

```
@@ -0,0 +1,99 @@
+services:
+  bellecour-fork:
+    restart: "no"
+    image: ghcr.io/foundry-rs/foundry:v1.0.0
+    entrypoint: anvil
+    command: "--host 0.0.0.0 --port 8545 --hardfork berlin --fork-url $BELLECOUR_FORK_URL --fork-block-number $BELLECOUR_FORK_BLOCK --chain-id 134 --gas-limit 6700000 --gas-price 0"
```

```suggestion
    command: "--host 0.0.0.0 --port 8545 --hardfork berlin --fork-url $BELLECOUR_FORK_URL --chain-id 134 --gas-limit 6700000 --gas-price 0"
```
Not working on my laptop locally, maybe due to macbook m3 chip

---

Seems to work now weird 🤔